hw3/HaHaFi/makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: test
+.PHONY: clean
+
+build_file_name=_matrix$(shell python3-config --extension-suffix)
+
+$(build_file_name): multiplication.cpp matrix.o
+	g++ -O3 -Wall -shared -std=c++11 -fPIC `python3-config --includes`  `python3 -m pybind11 --includes` \
+	multiplication.cpp matrix.o  -o $(build_file_name) -L/usr/lib/x86_64-linux-gnu -lblas -I/usr/include/mkl
```

Comment:
Let me check about it.

Reply:
Well, I use ldd to check original linking share library which is **libblas.so.3**.
I change makefile to linking **libmkl_rt.so**.
The result seems good at mkl function performance.

Reply:
Thanks for testing that.  This is good finding.  Since I do not have a bare metal with Linux it's a little bit hard for me to reproduce what you see.  But I am interested in what you have now.

If you have time, and are also interested in the runtime performance, perhaps we can discuss more.  We can follow up in https://discord.gg/6MAkFrD .  Ping me `@yyc` there.